bower.json

```diff
@@ -9,11 +9,13 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
+    "baguettebox.js": "~1.8",
     "bootstrap": "~3.3.7",
+    "html5shiv": "^3.7.3",
     "jquery": "<2.0.0",
     "jquery-colorbox": "~1",
-    "moment": "~2",
+    "katex": "^0.7.1",
```

Comment:
I'll take katex out and think on this.

We might be able to structure things to default to `USE_CDN = True` and only download/install node and the like if people specify `USE_CDN = False`. I'll think on that separately.